sema/commons/glob/globbery.py

```
@@ -0,0 +1,88 @@
+        """
+        Called when a directory is visited.
+        @param path: The directory path.
+        @param applying: A list of apply objects with matching globs.
+        """
+        pass
+
+
+def visitGlobPaths(
+    visitor: GlobMatchVisitor,
+    root: Path,
+    includes: List[str] = ["**/*"],
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)